test/data/.sgnd/config/genesis.json

```diff
@@ -179,25 +174,15 @@
       "redelegations": null,
       "exported": false
     },
-    "govern": {
-      "starting_proposal_id": "1",
-      "deposits": null,
-      "votes": null,
-      "proposals": null,
-      "deposit_params": {
-        "min_deposit": "1",
-        "max_deposit_period": "172800000000000",
-        "muted_duration": "10000000000"
-      },
+    "sync": {
+      "starting_change_id": "1",
+      "changes": null,
       "voting_params": {
-        "voting_period": "10000000000"
+        "voting_period": "15000000000"
```

Comment:
some of the genesis fields changed, plz restore the previous values back if the changes are unintended